src/components/Navigation/Navigation.scss

```diff
@@ -208,14 +208,14 @@ $disabled-fade: 0.6;
     @include recolor-icon(
       var(--p-icon-hovered),
       var(--p-override-transparent),
-      $filter-color: filter('action')
+      // $filter-color: filter('action')
```

Comment:
This will break channel icons because they get passed to the nav via graphql as images